config/routes.rb

```diff
@@ -6,7 +6,10 @@
 
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
-  delete '/logout', to: 'sessions#destroy'
 
+  resource :admin
+  
   root to: "items#index"
+
+  delete '/logout', to: 'sessions#destroy'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.